test/draw/RasterYuv420SemiplanarTest.ooc

```diff
@@ -26,6 +26,7 @@ RasterYuv420SemiplanarTest: class extends Fixture {
 			expect(rgbSample distance(ColorRgb new(0, 100, 200)), is less than(8.0f))
 			(yuvImage, rgbImage) free()
 		})
+		/* Why does this fail when not recycling memory?
```

Comment:
The bugs are deterministic on the same computer but changes when running on another computer.